src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -187,6 +187,7 @@
         <xsd:attribute name="enabled" type="xsd:boolean" />
         <xsd:attribute name="fallback" type="xsd:string" />
         <xsd:attribute name="logging" type="xsd:boolean" />
+        <xsd:attribute name="path" type="xsd:string" />
```

Comment:
this should not be an attribute but an element, as you need to be able to add multiple paths (see ``fallback``)